loaders/src/glTF/2.0/babylon.glTFLoader.ts

```diff
@@ -1739,6 +1760,9 @@ module BABYLON.GLTF2 {
                         }
                     }
                 }
+                else if (mesh.material instanceof Material) {

```

Comment:
All materials have Material in their prototype chain so this test will always be true